x-pack/plugins/security_solution/server/lib/detection_engine/scripts/roles_users/README.md

```diff
@@ -0,0 +1,13 @@
+1. When first starting up elastic, detections will not be available until you visit the page with a SOC Manager role or Platform Engineer role
+2. The T1 and T2 Analyst roles do not seem to have much of a functional difference but I created them as is.
```

Comment:
In the PR description `T1 Analyst` `Lists` is defined as `none`, same in the linked worksheet, so I think this is the only difference then? Might just be a typo swap between `Lists` and `Security Solution ML Results` as I think that should be `none`.

<p align="center">
  <img width="300" src="https://user-images.githubusercontent.com/2946766/99015311-edd89d00-2511-11eb-8eff-a2c3db01631f.png" />
</p>